docs/components/datetimepicker.md

```diff
@@ -0,0 +1,190 @@
+# DateTimePicker
+
+<div v-if="hasMounted">
+
+**KDateTimePicker** - A date and time selection tool, displayed inside a popover.
+
+## Examples
+
+### Custom + Relative date/time ranges
+
+Display both a calendar and relative time frames, passing in a preset week-long range.
+
+<KCard>
+  <template v-slot:body>
+    <KDateTimePicker
+      @changed="newVal => currentValue1 = newVal"
+      defaultMessage="Please select a time range"
+      :timePeriods="exampleTimeFrames"
+      :defaultCustom="defaultRangeWeek"
+      v-model="currentValue1"
+    />
+    <div class="mt-6">Emitted value: <pre>{{ currentValue1 }}</pre></div>
+  </template>
+</KCard>
+
+```html
+<KDateTimePicker 
+  @change="newVal => currentValue = newVal" 
+  :defaultCustom="defaultRangeWeek"
+  :timePeriods=[
+    {
+      section: 'Last',
+      values: [
+        { timeframeText: '15 minutes', timeframeLength: 60*15 },
+        ...
+      ]
+    },{
+      section: 'Previous',
+      values: [
+        { timeframeText: '15 minutes', timeframeLength: 60*15 },
```

Comment:
A previous timeframe needs a custom `start()` and `end()` can't just use a timeframeLength to figure it out